### ISSUE TITLE
[UXToolsGame | HoloLens 2] 3D Objects' coordinates are wrong after waking up from sleep mode

### ISSUE THREAD

**Issue by Zikun-Huang** [microsoft/MixedReality-UXTools-Unreal#32]
### Issue description
Every time wake up HoloLens 2 from a 10-to-30-minutes sleep, the placed 3D Objects will be shifted to a random place as a group (i.e. absolute coordinates are wrong, while relative coordinates are still correct). It looks like the origin of the "world" has moved to a new random point out of unknown reason.

**Before getting into sleep mode:**
![20210519_223658_HoloLens](https://user-images.githubusercontent.com/52370186/118927545-85f98800-b974-11eb-95b7-27829e117f28.jpg)
**After waking up from sleep mode:**
![20210519_225640_HoloLens](https://user-images.githubusercontent.com/52370186/118927582-96116780-b974-11eb-8119-3b943290a768.jpg)

### Steps for issue reproducing
1. Download [UXToolsGame for Unreal v0.11.0](https://github.com/microsoft/MixedReality-UXTools-Unreal/releases/download/v0.11.0/UXTGame-HoloLens.0.11.0.zip) and install it via Device Portal for HoloLens 2.
2. Open the application, click "Manipulator" which places several 3D cheeses in the room.
2. Single-press power button to get HoloLens 2 into sleep mode, then wait for 10-30 minutes.
3. Press power button to wake up HoloLens 2, observe 3D cheeses are in a new random place where relative positions are correct.
4. The issue happens intermittently but with high frequency (almost one in two runs)

### Troubleshooting we tried and more information
1. Environment info
- Windows OS 19041.1144 and 19041.1128 (found issue on both version)
- Unreal Engine 4.26.2
- UX Tools for Unreal 0.11.0
2. We tried to remove all hologram data and run eye calibration, but the issue persists.
3. We test with [MRTK Sample Hub](https://www.microsoft.com/en-us/p/mrtk-examples-hub/9mv8c39l2sj4?activetab=pivot:overviewtab) (based on Unity 3D), and there is no issue.
4. We also try with 2D applications, and no issue occurs.

**Comment by Zikun-Huang** [comment]
Upgrade to latest monthly release of Windows for Hologram (21H1 20346.1002) seems to fix the issue. Will conduct more tests to verify this.
Refer this doc for how-to upgrade if anyone is interesting on it.
https://docs.microsoft.com/en-us/hololens/hololens-recovery#clean-reflash-the-device
![image](https://user-images.githubusercontent.com/52370186/118954387-4b9de400-b990-11eb-918d-90f51d29c0f9.png)

**Comment by luis-valverde-ms** [comment]
Thanks for reporting the issue which such great detail and for following up @Zikun-Huang . @fieldsJacksonG , do you have an inkling as to what may have changed in Windows that affected just Unreal apps in this way?

**Comment by Zikun-Huang** [comment]
[New Update on issue]
After the upgrade, we found that the issue occurrence drops dramatically from 1/2 or 1/3 to 1/10 approx. 
Not sure if that means we are safe to say the issue gets fixed.

**Comment by luis-valverde-ms** [comment]
Thanks @Zikun-Huang . So you are still seeing the issue with the latest version of Windows, just less frequently? Are you using the Windows Mixed Reality plugin or the OpenXR one?

**Comment by Zikun-Huang** [comment]
> Thanks @Zikun-Huang . So you are still seeing the issue with the latest version of Windows, just less frequently? Are you using the Windows Mixed Reality plugin or the OpenXR one?

No worries Luis!
So yes, it is quite less frequent than that in previous OS version. I just simply download the UXToolsGame for Unreal v0.11.0 from Release of this Repo, and install it to HoloLens 2 via Device Portal, not went through any coding/build process. But I guess I am using Windows MR plugin?

**Comment by luis-valverde-ms** [comment]
Yes, that'd be WMR. @fieldsJacksonG is having a look on the engine side. We'll keep you posted.
